source/posix/system_info.c

```diff
@@ -313,8 +321,20 @@ void aws_backtrace_print(FILE *fp, void *call_site_data) {
         return;
     }
 
+    fprintf(fp, "################################################################################");
+    fprintf(fp, "Raw stacktrace:\n");
+    fprintf(fp, "################################################################################");
+    for (size_t frame_idx = 1; frame_idx < stack_depth; ++frame_idx) {
+        const char *symbol = symbols[frame_idx];
+        fprintf(fp, "%s\n", symbol);
+    }
```

Comment:
Can we do this one second? the resolved stack trace is likely to be more helpful, and should probably be printed first.